config/schema.yaml

```diff
@@ -1,1290 +1,156 @@
 schema:
   version: 0
-  dataUnits:
-    Camera:
-      doc: >
-        An entity that produces observations.  A Camera defines a set of
-        PhysicalFilters and Sensors and a numbering system for the Exposures
-        and Visits that represent observations with it.
-      link:
+  tables:
+    Dataset:
+      columns:
+      -
+        name: dataset_id
+        type: int
+        primary_key: true
+        nullable: false
+        doc: >
+          a unique autoincrement field used as part of the primary key for
+          dataset.
+      -
+        name: dataset_type_name
+        type: string
+        nullable: false
+        doc: >
+          the name of the datasettype associated with this dataset; a
+          reference to the datasettype table.
+      -
+        name: run_id
+        type: int
+        nullable: false
+        doc: >
+          the id of the run that produced this dataset, providing access to
+          coarse provenance information.
+      -
+        name: quantum_id
+        type: int
+        doc: >
+          the id of the quantum that produced this dataset, providing access
+          to fine-grained provenance information.
+          may be null for datasets not produced by running a supertask.
+      -
+        name: assembler
+        type: string
+        doc: >
+          fully-qualified name of an importable assembler object that can be
+          used to construct this dataset from its components.  null for
+          datasets that are not virtual composites.
```

Comment:
@pschella do you remember why this was?

Reply:
I don't think that code is there any more. Removed in 84970666d4e472dcf586c9a9be6bc12a058bec9f a week or so back. Assemblers are not in the schema any more.

Reply:
Note that @timj is right (@natelust, you can just remove `assembler` anywhere you see it in the schema), but the original comment was actually on the (not-shown-in-the-summary) lines below, because there wasn't a good diff line on which to attach that comment.

Reply:
@TallJimbo, @natelust because it was much easier and clearer to just put the link column descriptions here and have no special code. But if deemed desirable it could be auto-generated of course.